### PR TITLE
renamed DebugFile => DebugLog

### PR DESCRIPTION
**Pull request by kazimir-malevich**
- closes https://github.com/tag1consulting/goose/issues/462

### CODE REVIEW

File: src/docs/goose-book/src/config/defaults.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ For example, without any run-time options the following load test would automati
         )
         .set_default(GooseDefault::Host, "local.dev")?
         .set_default(GooseDefault::RequestsFile, "goose-requests.log")?
```
Comment (jeremyandrews) [inline src/docs/goose-book/src/config/defaults.md:69]: Can you also fix this? It should be [`GooseDefault::RequestLog`](https://docs.rs/goose/*/goose/config/enum.GooseDefault.html#variant.RequestLog)

File: src/docs/goose-book/src/config/defaults.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ The following defaults can be configured with a `&str`:
  - html-formatted report file name: `GooseDefault::ReportFile`
  - requests log file name: `GooseDefault::RequestsFile`
```
Comment (jeremyandrews) [inline src/docs/goose-book/src/config/defaults.md:21]: This should be [`GooseDefault::RequestLog`](https://docs.rs/goose/*/goose/config/enum.GooseDefault.html#variant.RequestLog)

File: src/docs/goose-book/src/config/defaults.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ The following defaults can be configured with a `&str`:
...
  - requests log file format: `GooseDefault::RequestsFormat`
```
Comment (jeremyandrews) [inline src/docs/goose-book/src/config/defaults.md:22]: This should be [`GooseDefault::RequestFormat`](https://docs.rs/goose/*/goose/config/enum.GooseDefault.html#variant.RequestFormat)

File: src/docs/goose-book/src/config/defaults.md [hunk 4]
```diff
@@ -79,4 +79,4 @@ For example, without any run-time options the following load test would automati
     Ok(())
 ```
 
-Find a complete list of all configuration options that can be configured with custom defaults [in the developer documentation](https://docs.rs/goose/*/goose/config/enum.GooseDefault.html), as well as complete details on [how to configure defaults](https://docs.rs/goose/*/goose/config/trait.GooseDefaultType.html).
\ No newline at end of file
+Find a complete list of all configuration options that can be configured with custom defaults [in the developer documentation](https://docs.rs/goose/*/goose/config/enum.GooseDefault.html), as well as complete details on [how to configure defaults](https://docs.rs/goose/*/goose/config/trait.GooseDefaultType.html).
```
Comment (jeremyandrews) [inline src/docs/goose-book/src/config/defaults.md:82]: What's changed here? I'm not seeing a difference in the Github UI?

**Review by jeremyandrews — CHANGES_REQUESTED** [review]
Thank you! There's a few similar documentation issues on this page that should also be fixed.

It also looks like your PR needs to be rebased?

File: src/docs/goose-book/src/config/defaults.md (hunk 4 above)
Comment (kazimir-malevich) [reply src/docs/goose-book/src/config/defaults.md:82]: That's confusing me.

File: src/docs/goose-book/src/config/defaults.md (hunk 1 above)
Comment (kazimir-malevich) [reply src/docs/goose-book/src/config/defaults.md:69]: I might have to re-fork, create a branch and add in my changes and all your suggestions. Would this be a problem? Then link the new  PR to this one? Until I figure out the correct way of doing this? Be tomorrow, now though.

**Pull request closed without merging**